### ISSUE TITLE
Error when the vcf contains no sample.

### ISSUE THREAD

**Issue by susannasiebert** [griffithlab/pVAC-Seq#236]
```
Executing MHC Class I predictions

Converting VCF to TSV

Traceback (most recent call last):

  File "/soft/python/3/bin/pvacseq", line 11, in <module>

    sys.exit(main())

  File "/soft/python/3/lib/python3.5/site-packages/pvacseq/pvacseq.py", line 84,                                          in main

    args[0].func.main(args[1])

  File "/soft/python/3/lib/python3.5/site-packages/pvacseq/lib/main.py", line 28                                         2, in main

    pipeline.execute()

  File "/soft/python/3/lib/python3.5/site-packages/pvacseq/lib/pipeline.py", lin                                         e 245, in execute

    self.convert_vcf()

  File "/soft/python/3/lib/python3.5/site-packages/pvacseq/lib/pipeline.py", lin                                         e 88, in convert_vcf

    lib.convert_vcf.main(convert_params)

  File "/soft/python/3/lib/python3.5/site-packages/pvacseq/lib/convert_vcf.py",                                          line 189, in main

    genotype = entry.genotype(vcf_reader.samples[0])

IndexError: list index out of range
```

**Comment by susannasiebert** [comment]
@chrisamiller @jhundal How should we handle vcf files with no sample genotype field? In that case should we error out totally, basically requiring exactly one sample, or should we process all variants? Normally we check the sample's genotype to ensure that the variant was called before including it in the processing. But when no sample genotype information is given, this is obviously not possible and we get the above error.

**Comment by chrisamiller** [comment]
If it's not overly burdensome to implement, I vote for writing out a warning message and processing all the variants. In general, I lean towards making it as easy as possible to get data into the pipeline.

**Comment by susannasiebert** [comment]
It's not difficult at all. I will make a pull request.

**Comment by susannasiebert** [comment]
This error was fixed in version 4.0.4.
